index.js

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env node
+const child_process = require("child_process");
+const minimist = require("minimist");
+
+const {runChecks, getCommands} = require("./src/utils");
+
+var argv = require('minimist')(process.argv.slice(2));
+
+try {
+    runChecks();
+    const commands = getCommands();
+    if (commands.length === 0) {
+        console.log("all packages up to date");
+        process.exit(0);
+    }
+
+    for (const command of commands) {
+        if (!argv["dry-run"]) {
+            console.log(`running: ${command}`);
+            child_process.execSync(command, {encoding: "utf8"});
+        } else {
+            console.log(`would run: ${command}`);
+        }
+    }
```

Comment:
No exit code for a successful run?

Reply:
I'm going to add a `process.exit(0);` but it has to come right at the end of the `try` block.

Reply:
Or after the catch?